generators/client/templates/react/package.json.ejs

```diff
@@ -35,9 +35,9 @@ limitations under the License.
     <%_ if (clientTheme !== 'none') { _%>
     "bootswatch": "4.3.1",
     <%_ } _%>
+    "dayjs": "1.9.3",
```

Comment:
Move the version to package.json, so it will be managed by dependabot.

Reply:
In which package.json should I put it? Should I completely remove it from from the other package.json.ejs, and how will it work?
Sorry, I would need explainations here :D

Reply:
Moved to angular.